server/src/main/java/org/elasticsearch/common/inject/internal/FirstStringUpper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2006 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.elasticsearch.common.inject.internal;
+
+/**
+ * String utilities.
+ *
+ * @author crazybob@google.com (Bob Lee)
+ */
+public class FirstStringUpper {
+    private FirstStringUpper() {}
+
+    /**
+     * Returns a string that is equivalent to the specified string with its
+     * first character converted to uppercase as by {@link String#toUpperCase}.
+     * The returned string will have the same value as the specified string if
+     * its first character is non-alphabetic, if its first character is already
+     * uppercase, or if the specified string is of length 0.
+     * <p>
+     * For example:
+     * <pre>
+     *    capitalize("foo bar").equals("Foo bar");
+     *    capitalize("2b or not 2b").equals("2b or not 2b")
+     *    capitalize("Foo bar").equals("Foo bar");
+     *    capitalize("").equals("");
+     * </pre>
+     *
+     * @param str the string whose first character is to be uppercased
+     * @return a string equivalent to {@code s} with its first character
+     *         converted to FirstStringUpper
+     * @throws NullPointerException if {@code s} is null
+     */
+    public static String capitalize(String str) {

```

Comment:
This method is in a single class - is there an existing method in another class that has similar functionality, that could be used instead, so allowing this class (and the `internal.Strings` class) to be removed entirely?